collector/src/androidTest/java/com/bitmovin/analytics/data/EventDataTest.java

```diff
@@ -39,4 +42,14 @@ public void serializeTest() throws Exception {
         System.out.println(json);
         Assert.assertEquals(json, expected);
     }
+
+    // Test isn't running because I can't get android to compile the apk for the tests :/
```

Comment:
I guess you're also running into the `Cannot fit requested classes in a single dex file (# methods: 68846 > 65536)` error?

Add this to `android.defaultConfig` section of the `build.gradle` file, then it should work:
```
        testOptions {
            multiDexEnabled true
        }
```